lab-cameron/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "single-resource-express-api",
+  "version": "0.0.1",
+  "description": "lab 11",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "lint": "eslint ./server.js",
+    "test": "mocha",
+    "start": "node server.js",
+    "watch": "nodemon server.js",
+    "build": "npm run lint && npm run test"
+  },
+  "keywords": [
+    "express",
+    "API",
+    "node"
+  ],
+  "author": "Cameron Bacon",
+  "license": "MIT",
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "chai-http": "^3.0.0",
+    "eslint": "^3.19.0",
+    "mocha": "^3.3.0",
+    "morgan": "^1.8.1"
+  },
+  "dependencies": {
```

Comment:
**TODO FOR LOST POINTS**

Missing ``express`` as a dependency. This breaks everything. I'll still install myself and review your code, but be mindful that this is normally an automatic 0.